### PR TITLE
Add colors to gobusterdir (#115)

### PR DESCRIPTION
**Pull request by eranchetz**
This relates to #115.
If there is a more efficient way to determine if a request is 200, I would be more than happy to make the changes.

### CODE REVIEW

File: gobusterdir/gobusterdir.go [hunk 1]
```diff
@@ -87,11 +89,11 @@ func (d GobusterDir) ResultToString(g *libgobuster.Gobuster, r *libgobuster.Resu
 	// Prefix if we're in verbose mode
 	if g.Opts.Verbose {
 		if g.Opts.StatusCodesParsed.Contains(r.Status) {
-			if _, err := fmt.Fprintf(buf, "Found: "); err != nil {
+			if _, err := fmt.Fprintf(buf, Sprintf(Green("Found: "))); err != nil {
 				return nil, err
 			}
 		} else {
-			if _, err := fmt.Fprintf(buf, "Missed: "); err != nil {
+			if _, err := fmt.Fprintf(buf, Sprintf(Red("Missed: "))); err != nil {
```
Comment (OJ) [inline gobusterdir/gobusterdir.go]: I think I'd rather see `Missed` outputted in orange or yellow. It's not exactly an error.

File: gobusterdir/gobusterdir.go [hunk 2]
```diff
@@ -112,7 +114,11 @@ func (d GobusterDir) ResultToString(g *libgobuster.Gobuster, r *libgobuster.Resu
 		}
 
 		if !g.Opts.NoStatus {
-			if _, err := fmt.Fprintf(buf, " (Status: %d)", r.Status); err != nil {
+			statusWithColor := Sprintf(Brown(r.Status))
+			if r.Status == 200 {
+				statusWithColor = Sprintf(Green(r.Status))
```
Comment (OJ) [inline gobusterdir/gobusterdir.go]: I think it's unnecessary to have different colours specifically for 200. If you get a hit, based on the requested status codes, that's success and should be green.

**Comment by OJ** [comment]
Also, can you please not submit pull requests from your master branch in future? Best to do it from a feature branch.

If you could rebase this against the v3.0-working branch also that'd be super helpful. Thanks!

**Comment by Anon-Exploiter** [comment]
@OJ @eranchetz any progress on the **colored output** of gobuster? :)

**Comment by eranchetz** [comment]
@Anon-Exploiter this have become more complicated than I originally thought.
I think I will forgo this :(

**Pull request closed without merging**